py2many/python_transformer.py

```diff
@@ -0,0 +1,20 @@
+import ast
+
+from .clike import CLikeTranspiler
+
+
+class PythonTranspiler(CLikeTranspiler):
+    NAME = "python"
+
+    def visit(self, tree):
+        return ast.unparse(tree) + "\nif __name__ == '__main__': main(0, sys.argv)"
```

Comment:
this bit needs to be rewritten to use https://github.com/adsharma/py2many/pull/349

`ast.unparse` is py3.9+, but I expect to be able to find a good backport that can be used to support py3.8